mypy/build.py

```diff
@@ -868,7 +870,46 @@ def is_file(path: str) -> bool:
     return res
 
 
-def find_module(id: str, lib_path_arg: Iterable[str]) -> Optional[str]:
+USER_SITE_PACKAGES = \
+    'from __future__ import print_function; import site; print(site.getusersitepackages());' \
+    'print(*site.getsitepackages(), sep="\\n")'
```

Comment:
This fails in the corner-case where the paths contain newlines(!)

Better would be to do

```
print(repr(site.getsitepackages() + [site.getusersitepackages()]))
```
and then read it with `ast.literal_eval`

Reply:
I also worry that you might have a different between the stdout encoding and the default `.decode()` decoding, which using `repr`/`ast.literal_eval` or maybe even `pickle` would resolve.

Are site package paths unicode, bytes, or either?

Reply:
Should be `str`, for each meaning in 2 and 3, so your recommendation is well taken.